BRAINSCommonLib/DWIMetaDataDictionaryValidator.cxx

```diff
@@ -229,7 +229,7 @@ DWIMetaDataDictionaryValidator::SetBValue(const double bvalue)
 }
 
 std::string
-DWIMetaDataDictionaryValidator::GetIndexedKeyString(const std::string base_key_name, const size_t index) const
+DWIMetaDataDictionaryValidator::GetIndexedKeyString(const std::string & base_key_name, const size_t index) const
```

Comment:
Should `const size_t index` be passed by reference?

Reply:
The performance-unnecessary-value-param clang-tidy check only searches for parameters that are expensive to copy.  Since `size_t` is just an `unsigned long`, it is not expensive to copy so converting it to pass-by-reference wouldn't increase efficiency. We could make the change for consistency if desired.

Reply:
I don't think it would be necessary to pass the unsigned long by reference - I just wanted to make sure it did not get passed over.